any_parser/any_parser.py

```diff
@@ -137,6 +136,62 @@ def extract(
         else:
             return f"Error: {response.status_code} {response.text}", None
 
+    def extract_json(
```

Comment:
This looks to me a very bad name. Especially, extract required no prompt while extract_json requires prompt. Then, suddently, you start to return json. Logically, I do not know why an extract_instruction starts to get this extract to extract_json. You should re-consider on naming this.

Reply:
updated to `extract_key_value`